samples/esb/esb_monitor/README.rst

```diff
@@ -0,0 +1,87 @@
+.. _esb_monitor:
+
+Enhanced ShockBurst: Monitor
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Monitor sample shows how to use the :ref:`ug_esb` protocol in Monitor mode.
+It demonstrates how to configure the Enhanced ShockBurst protocol to receive all the traffic generated on the configured channel and pipes.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+The sample consist of one receiver configured as a monitor that uses the :ref:`esb_README` library.
+After building and programming each sample on an nRF54 Series DK, you can monitor the traffic generated by the kits programmed with the :ref:`Transmitter <esb_ptx>` and :ref:`Receiver <esb_prx>` samples, respectively.
+You can see the received traffic in real time using the `Serial Terminal app`_.
+Successful monitoring is also indicated by the LEDs that should be in sync on all kits.
+
+User interface
+***************
+
+All LEDs:
+
+   Indicate that packets are received.
+   The first four packets turn on the LEDs sequentially.
+   The next four packets turn them off again in the same order.
```

Comment:
May be worth mentioning that this logic is aligned with the other samples in NCS: esb_prx and esb_ptx as it checks the payloads transmitted by PRX and PTX, which are sequential numbers. It won't work that way in a general case and in theory monitor may listen to more than 2 ESB nodes, then the LED behavior will differ.